Dockerfile

```diff
@@ -91,7 +91,7 @@ RUN pip install -r requirements-dev.txt
 RUN pip install tox==2.1.1
 
 ADD . /code/
-RUN python setup.py install
+RUN pip install --no-deps -e /code
```

Comment:
This change makes it so that dependencies from `setup.py` aren't reinstalled, and so that files aren't copied. It just creates a link to the source files in `/code`, so this build step is faster.